apps/meeting/scripts/update-react-library.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+/**
+ * This script clones and installs the latest code from amazon-chime-sdk-component-library-react
+ * GitHub repository when in meeting-dev branch.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { logger, spawnOrFail } = require('./utilities');
+const rimraf = require('rimraf');
+ 
+const currentBranch = spawnOrFail('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
+if (currentBranch === 'main') {
+  logger.log(`current branch is ${currentBranch}, using latest react library from NPM`);
+  return;
+}
+
+if (currentBranch === 'meeting-dev') {
+  logger.log(`current branch is ${currentBranch}, using react library from GitHub`);
+  const package = 'amazon-chime-sdk-component-library-react';
+  const dir = `${package}-github`;
+  const packageGitUrl = 'https://github.com/aws/amazon-chime-sdk-component-library-react.git';
+  
+  rimraf.sync(dir);
+  if (!fs.existsSync(dir)) {
+    logger.log(`Creating a temp directory ${dir}`);
+    fs.mkdirSync(dir);
+  }
+  
+  process.chdir(path.join(__dirname, `../${dir}`));
+  logger.log(`Cloning ${package} to update the existing installation.`);
+  spawnOrFail('git', [`clone ${packageGitUrl}`]);
+  
+  process.chdir(path.join(__dirname, `../${dir}/${package}`));
+  
+  logger.log(`Running npm install in cloned ${package}`);
+  spawnOrFail('npm', ['install']);
```

Comment:
Not sure if we need to `install` first, since every time I just build and pack, and it works.